TrendMicroVisionOne/action_terminate_process.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Terminate a process",
+  "description": "Terminate a process that is running on a list of endpoints",
+  "uuid": "903e85fd-05d6-524e-941d-adfaff02cb8c",
+  "docker_parameters": "action_terminate_process",
+  "arguments": {
+    "type": "object",
+    "required": ["agent_guids", "process_id"],
+    "properties": {
+      "agent_guids": {
+        "description": "The identifiers of the endpoints",
+        "type": "array",
+        "minItems": 1,
+        "items": {
+          "type": "string"
+        }
+      },
+      "process_id": {
+        "description": "Process ID",
+        "type": "integer"
+      },
```

Comment:
I suggest grouping the agent uuid with the process id as a same executable will have different process id on different machine.